simulation_parameters/Constants.cs

```diff
@@ -504,7 +504,12 @@ public static class Constants
     /// <summary>
     ///   Delete a max of this many entities per step to reduce lag from deleting tons of entities at once.
     /// </summary>
-    public const int MAX_DESPAWNS_PER_FRAME = 1;
+    public const int MAX_DESPAWNS_PER_FRAME = 2;
```

Comment:
Migrating happens maybe once every 5 minutes, and during the editor exiting loading screen. So that causing a lag spike there is not a problem. However during gameplay random lag spikes should be avoided.

Unless loading older saves the entity count shouldn't get out of hand in the first place.

Reply:
All right. How much lag does despawning cause? In my previous testing I experienced some jittering in FPS but I couldn't tell if it was from spawning or despawning.

I'll try testing this branch and see how it feels.